deps.edn

```diff
@@ -34,63 +37,52 @@
                  :main-opts ["-e" "(require 'hashp.core)"]}
 
            :test {:extra-paths ["test"]
-                  :extra-deps {lambdaisland/kaocha         {:mvn/version "1.70.1086"}
-                               lambdaisland/kaocha-cljs    {:mvn/version "1.0.107"}
+                  :extra-deps {lambdaisland/kaocha         {:mvn/version "1.71.1119"}
+                               lambdaisland/kaocha-cljs    {:mvn/version "1.4.130"}
                                org.clojure/test.check      {:mvn/version "1.1.1"}}}
 
-           :repl {:extra-deps {cider/cider-nrepl           {:mvn/version "0.28.3"}
-                               nrepl/nrepl                 {:mvn/version "0.9.0"}
-                               org.clojure/tools.namespace {:mvn/version "1.2.0"}}
+           :repl {:extra-deps {cider/cider-nrepl           {:mvn/version "0.28.7"}
+                               nrepl/nrepl                 {:mvn/version "1.0.0"}
+                               org.clojure/tools.namespace {:mvn/version "1.3.0"}}
                   :main-opts ["-m" "nrepl.cmdline" "--middleware" "[cider.nrepl/cider-middleware]"]}
```

Comment:
I'd prefer to reduce the amount of aliases here, especially the ones that are dependent on the personal setup like starting a repl. people can have a global `deps.edn` in `~/.clojure` folder. But maybe that's just me....

Reply:
Ah, I actually thought you were using this alias. I'd be happy dropping it.

Reply:
I removed it.